### ISSUE TITLE
Composer installation fails

### ISSUE THREAD

**Issue by kristoftorfs** [megawubs/trakt-api-wrapper#7]
Composer installation fails due to unsatisfiable dependencies:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for wubs/trakt dev-master -> satisfiable by wubs/trakt[dev-master].
    - wubs/trakt dev-master requires nesbot/carbon dev-master -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://groups.google.com/d/topic/composer-dev/_g3ASeIFlrc/discussion> for more details.

Read <http://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

**Comment by csrbrito** [comment]
I got the same problem but i made it by change some settings on composer.json ( load the version that the scripts requires with another alias ), later a will tell you the  workarround for that, but yes, its not stable enouph. I'm making my own fork version on this with some improvements and stability.

**Comment by megawubs** [comment]
I noted some changes in how composer worked during my work this week. For
now, I think adding a tag should be enough. But I don't know if I want to
give it a tag yet, as it's still not the way I want it to work.

Op do 23 apr. 2015 10:08 schreef csrbrito notifications@github.com:

**Comment by makop79** [comment]
I had the same problem during the installation - installing a git client solved it for me. After that everything went fine.

**Comment by joshlopes** [comment]
you need to set dependencies in such cases, for that case just add as required that dev version of nesbot and you should be fine.

```
require: {
    ....
    "nesbot/carbon": "dev-master"
} 
```

or run the command:

```
composer require "nesbot/carbon:dev-master"
```

Do the follow for all dependencies that are not match. Remember that composer will choose @stable over @dev if your minimun-stable is not set

**Comment by dogrocker** [comment]
Got an error too, When I trying to install on Laravel 4.2

```
sudo composer require "wubs/trakt:dev-master"
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Can only install one of: guzzlehttp/guzzle[5.3.0, 4.1.7].
    - Can only install one of: guzzlehttp/guzzle[5.3.0, 4.1.7].
    - Can only install one of: guzzlehttp/guzzle[5.3.0, 4.1.7].
    - wubs/trakt dev-master requires guzzlehttp/guzzle 5.3.0 -> satisfiable by guzzlehttp/guzzle[5.3.0].
    - Installation request for wubs/trakt dev-master -> satisfiable by wubs/trakt[dev-master].
    - Installation request for guzzlehttp/guzzle == 4.1.7.0 -> satisfiable by guzzlehttp/guzzle[4.1.7].


Installation failed, reverting ./composer.json to its original content.
```

```
sudo composer require wubs/trakt



  [InvalidArgumentException]
  Could not find package wubs/trakt at any version for your minimum-stability (stable). Check
   the package spelling or your minimum-stability



require [--dev] [--prefer-source] [--prefer-dist] [--no-progress] [--no-update] [--update-no-dev] [--update-with-dependencies] [--ignore-platform-reqs] [--sort-packages] [packages1] ... [packagesN]
```

**Comment by megawubs** [comment]
I'll look into this

Op vr 12 jun. 2015 12:08 schreef Kanin Peanviriyakulkit <
notifications@github.com>:

**Comment by megawubs** [comment]
you have to do `composer require wubs/trakt dev-master`. I'll update the docs.

Do you have another library that requires guzzle?

**Comment by dogrocker** [comment]
@megawubs Thanks, for fast reply.
I try to use `composer require wubs/trakt dev-master`

it got the same error.

```
sudo composer require wubs/trakt dev-master
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Can only install one of: guzzlehttp/guzzle[5.3.0, 4.1.7].
    - Can only install one of: guzzlehttp/guzzle[5.3.0, 4.1.7].
    - Can only install one of: guzzlehttp/guzzle[5.3.0, 4.1.7].
    - wubs/trakt dev-master requires guzzlehttp/guzzle 5.3.0 -> satisfiable by guzzlehttp/guzzle[5.3.0].
    - Installation request for wubs/trakt dev-master -> satisfiable by wubs/trakt[dev-master].
    - Installation request for guzzlehttp/guzzle == 4.1.7.0 -> satisfiable by guzzlehttp/guzzle[4.1.7].


Installation failed, reverting ./composer.json to its original content.
```

This is my composer.json

```
"require": {
        "raveren/kint": "dev-master",
        "laravel/framework": "4.2.*",
        "facebook/php-sdk-v4" : "4.0.*",
        "nesbot/Carbon": "*",
        "fabpot/goutte": "~2.0",
        "pingpong/trusty": "1.0.*",
        "jenssegers/agent": "*",
        "hieu-le/active": "~1.0",
        "league/fractal": "0.9.*",
        "intervention/image": "2.*",
        "liebig/cron": "dev-master"
    },
```

I think that `fabpot/goutte` use guzzle 4\* library.

**Comment by megawubs** [comment]
Should be fixed with this commit 3feeae6935fd3e3a2c481d14ff1efcfc156161b0
